### PR TITLE
fix(one_dashboard): added raw configuration properties to one dashboard resource

### PR DESCRIPTION
**Pull request by RavitejaSurampudi**
# Description

Please include a summary of the change and which issue is fixed (if relevant).

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

Please delete options that are not relevant.

- [x] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes. Go [here](https://github.com/newrelic/terraform-provider-newrelic#testing) for instructions on running tests locally.

## How to test this change?

Please describe how to test your changes. Include any relevant steps in the UI, HCL file(s), commands, etc

- Step 1
- Step 2
- etc

### CODE REVIEW

File: website/docs/r/one_dashboard.html.markdown [hunk 1]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
   * `width` - (Optional) Width of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `4`.
   * `height` - (Optional) Height of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `3`.
   * `ignore_time_range` - (Optional) With this turned on, the time range in this query will override the time picker on dashboards and other pages. Defaults to `false`.
+  * `facet_show_other_series` - (Optional) With this turned on, this customization allows you to have the Other groups option visible or not if a facets ona query returns are more than 2000 for bar charts, pie charts, tables, the Other groups facet aggregates the rest of the facets. Defaults to `false`
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:244]: (Optional) Enable or disable the `Other` group in visualisations. The other group is used if a facet on a query returns more than 2000 items for bar charts, pie charts, and tables. The other group aggregates the rest of the facets. Defaults to `false`

File: website/docs/r/one_dashboard.html.markdown [hunk 2]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
...
+  * `y_axis_left_min`, `y_axis_left_max` - (Optional) Adjust the Y axis to display the data within certain values by setting a minimum and maximum value for the axis for line charts and area charts. If no customization option is selected, dashboards automatically displays the full Y axis from 0 to the top value plus a margin.
+  * `legend_enabled` - (Optional) wWith this turned on, the legend will be displayed. Defaults to `false`.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:246]: Typo `wWith` and is this correct? Don't we show the legend by default? So the default would be true?

File: website/docs/r/one_dashboard.html.markdown [hunk 3]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
...
+  * `null_values` - (Optional) A nested block that describes a Null Values.  See [Nested Null Values blocks](#nested-null-values-blocks) below for details.
+  * `units` - (Optional) A nested block that describes a Null Values.  See [Nested Units blocks](#nested-units-blocks) below for details.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:248]: Typo `describes a Null Values`

File: website/docs/r/one_dashboard.html.markdown [hunk 4]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
...
+  * `colors` - (Optional) A nested block that describes a Null Values.  See [Nested Colors blocks](#nested-colors-blocks) below for details.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:249]: Typo `describes a Null Values`

File: website/docs/r/one_dashboard.html.markdown [hunk 5]
```diff
@@ -260,6 +338,27 @@ The following arguments are supported:
   * `title` - (Optional) A human-friendly display string for this value.
   * `value` - (Required) A possible variable value
 
+### Nested `Null Values` blocks
+
+The following arguments are supported:
+
+* `null_value` - (Optional) Choose an option in displaying null values. Accepted values are `deafult`, `remove`, `preserve`, or `zero`.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:345]: Small corrections
`Choose an option for displaying null values.`

`deafult`

File: website/docs/r/one_dashboard.html.markdown [hunk 6]
```diff
@@ -260,6 +338,27 @@ The following arguments are supported:
...
+* `series_overrides` - (Optional) A Nested block which will take two string attributes `null_value` and `series_name`. This nested block is used to customize null values of individual series.
+
+### Nested `Units` blocks
+
+The following arguments are supported:
+
+* `unit` - (Optional) Choose a unit to customize the unit on your Y axis and in each of your series.
+* `series_overrides` - (Optional) A Nested block which will take two string attributes `null_value` and `series_name`. This nested block is used to customize null values of individual series.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:353]: `null_value` is that correct?

**Review by kidk — CHANGES_REQUESTED** [review]

File: website/docs/r/one_dashboard.html.markdown (hunk 1 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated

File: website/docs/r/one_dashboard.html.markdown (hunk 2 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: corrected typo

File: website/docs/r/one_dashboard.html.markdown (hunk 3 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated to units

File: website/docs/r/one_dashboard.html.markdown (hunk 4 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated to colors

File: website/docs/r/one_dashboard.html.markdown (hunk 5 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated

File: website/docs/r/one_dashboard.html.markdown (hunk 6 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated to unit

File: website/docs/r/one_dashboard.html.markdown (hunk 2 above)
Comment (kidk) [reply website/docs/r/one_dashboard.html.markdown]: The default for `legend_enabled` is true, not false. Just double checked in UI.

File: website/docs/r/one_dashboard.html.markdown [hunk 7]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
   * `width` - (Optional) Width of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `4`.
   * `height` - (Optional) Height of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `3`.
   * `ignore_time_range` - (Optional) With this turned on, the time range in this query will override the time picker on dashboards and other pages. Defaults to `false`.
+  * `facet_show_other_series` - (Optional) Enable or disable the Other group in visualisations. The other group is used if a facet on a query returns more than 2000 items for bar charts, pie charts, and tables. The other group aggregates the rest of the facets. Defaults to `false`
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:244]: I think the default should also be true here, can you double check

File: website/docs/r/one_dashboard.html.markdown [hunk 8]
```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
...
+  * `y_axis_left_min`, `y_axis_left_max` - (Optional) Adjust the Y axis to display the data within certain values by setting a minimum and maximum value for the axis for line charts and area charts. If no customization option is selected, dashboards automatically displays the full Y axis from 0 to the top value plus a margin.
+  * `legend_enabled` - (Optional) With this turned on, the legend will be displayed. Defaults to `true`.
+  * `null_values` - (Optional) A nested block that describes a Null Values.  See [Nested Null Values blocks](#nested-null-values-blocks) below for details.
+  * `units` - (Optional) A nested block that describes  units on your Y axis.  See [Nested Units blocks](#nested-units-blocks) below for details.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:248]: The spaces `  ` after `describes

Comment (kidk) [reply website/docs/r/one_dashboard.html.markdown:248]: Same for after period

File: website/docs/r/one_dashboard.html.markdown [hunk 9]
```diff
@@ -260,6 +338,27 @@ The following arguments are supported:
   * `title` - (Optional) A human-friendly display string for this value.
   * `value` - (Required) A possible variable value
 
+### Nested `Null Values` blocks
+
+The following arguments are supported:
+
+* `null_value` -  Choose an option in displaying null values. Accepted values are `deafult`, `remove`, `preserve`, or `zero`.
```
Comment (kidk) [inline website/docs/r/one_dashboard.html.markdown:345]: `deafult`

**Review by kidk — CHANGES_REQUESTED** [review]

File: website/docs/r/one_dashboard.html.markdown (hunk 9 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown:345]: @kidk do you want me to add the default value for this attribute ?

File: website/docs/r/one_dashboard.html.markdown (hunk 7 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown:244]: by default it is false.

File: website/docs/r/one_dashboard.html.markdown (hunk 8 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated.

File: website/docs/r/one_dashboard.html.markdown (hunk 9 above)
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated.

File: website/docs/r/one_dashboard.html.markdown [hunk 10]
```diff
@@ -260,6 +338,27 @@ The following arguments are supported:
...
+* `series_overrides` - (Optional) A Nested block which will take two string attributes `null_value` and `series_name`. This nested block is used to customize null values of individual series.
+
+### Nested `Units` blocks
+
+The following arguments are supported:
+
+* `unit` - (Optional) Choose a unit to customize the unit on your Y axis and in each of your series.
+* `series_overrides` - (Optional) A Nested block which will take two string attributes `unit` and `series_name`. This nested block is used to customize null values of individual series.
```
[the remark this answers is not in the document]
Comment (RavitejaSurampudi) [reply website/docs/r/one_dashboard.html.markdown]: updated

**Review by NSSPKrishna — APPROVED** [review]

**Pull request merged**